src/Compilers/Core/Portable/Syntax/SyntaxListBuilder.cs

```diff
@@ -153,7 +155,7 @@ public bool Any(int kind)
         {
             for (int i = 0; i < Count; i++)
             {
-                if (_nodes[i].Value.RawKind == kind)
+                if (_nodes[i].Value!.RawKind == kind)
```

Comment:
❔ Is it possible for this to reach external code and still be null?

Reply:
Not that I'm aware of. It's an implementation detail of this type.

---
In reply to: [362963462](https://github.com/dotnet/roslyn/pull/40729#discussion_r362963462) [](ancestors = 362963462)